task-requests/script.js

```diff
@@ -60,22 +60,25 @@ async function getTaskRequests(query = {}, nextLink) {
 
     if (res.status === 401) {
       showMessage('ERROR', ErrorMessages.UNAUTHENTICATED);
-      return;
+      return { data: [] };
```

Comment:
did you check via unit test

Reply:
Yes sir, I have attached the test results